### PR TITLE
API Stubs

### PR DESCRIPTION
**Pull request by SergioMorchon**
Built on top of #49

### CODE REVIEW

File: app/src/api/dev-config.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const USE_STUBS = false;
```
Comment (SergioMorchon) [inline app/src/api/dev-config.ts:1]: Locally change this to `true` for using hardcoded stubs. It's a rude mechanism, but dealing with `react-tscripts`'s and its inner usage of `dotenv` made me waste a lot of time without any final clue 😣

File: backend/controllers/utils.js [hunk 2]
```diff
@@ -24,8 +24,8 @@ const utils = {
             deliverFee: FEES.deliverFee,
             shopperFee: FEES.shopperFee,
             deliveryDiscount: FEES.discount,
-            finalDeliverFee: FEES.discount ? FEES.deliverFee * FEES.discount : FEES.deliverFee,
-            finalShopperFee: FEES.discount ? FEES.shopperFee * FEES.discount : FEES.shopperFee,
```
Comment (SergioMorchon) [inline backend/controllers/utils.js:28]: I just saw this when creating the payment stub.
The greater the discount, the smaller the final price.

Comment (danicg) [reply backend/controllers/utils.js:28]: 👍

**Review by danicg — APPROVED** [review]
👍

**Pull request merged**